packages/web-react/src/components/Button/demo/createButtonDemo.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Button } from '../Button';
+import { Icon } from '../../Icon';
+import { ButtonBaseProps } from '../../../types';
+
+const createButtonDemo = (labels: [string, string], props?: ButtonBaseProps) => {
+  const sizes = ['small', 'medium', 'large'];
```

Comment:
We have dictionariers/constants for this, so please use them ;-)

Reply:
Good idea